services/dask-sidecar/src/simcore_service_dask_sidecar/computational_sidecar/docker_utils.py

```diff
@@ -44,33 +51,40 @@
 
 
 async def create_container_config(
+    *,
     docker_registry: str,
-    service_key: str,
-    service_version: str,
-    command: list[str],
+    image: ContainerImage,
+    tag: ContainerTag,
+    command: ContainerCommands,
     comp_volume_mount_point: str,
     boot_mode: BootMode,
     task_max_resources: dict[str, Any],
+    envs: ContainerEnvsDict,
+    labels: ContainerLabelsDict,
 ) -> DockerContainerConfig:
     nano_cpus_limit = int(task_max_resources.get("CPU", 1) * 1e9)
     memory_limit = ByteSize(task_max_resources.get("RAM", 1024**3))
+    env_variables = [
+        f"{name.upper()}_FOLDER=/{name}s"
```

Comment:
MINOR: for three items i find more clear to write them exclicitly than a for loop :-)

Reply:
was a copy/paste of code from before. but ok.